### PR TITLE
Added Changelog changes for Batch consumer related fixes

### PR DESCRIPTION
**Pull request by pranavrth**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -40,6 +40,28 @@ librdkafka v2.1.0 is a feature release:
...
+   used for a partition.
+
+
+## Known Issues
+
+### Consume Batch API
+
+ * When `rd_kafka_consume_batch()` and `rd_kafka_consume_batch_queue()` APIs are used with
+   any of the **seek**, **pause**, **resume** or **rebalancing** operation, `on_consume`
+   iterceptors might be called incorrectly (maybe multiple times) for not consumed message.
```
Comment (emasab) [inline CHANGELOG.md:59]: "not consumed messages"?

Comment (emasab) [reply CHANGELOG.md:59]: "interceptors"

File: CHANGELOG.md [hunk 2]
```diff
@@ -40,6 +40,28 @@ librdkafka v2.1.0 is a feature release:
...
+ * Using multiple instances of `rd_kafka_consume_batch()` and `rd_kafka_consume_batch_queue()`
+   APIs in conjunction with any of the **seek**, **pause**, **resume** or **rebalancing**
+   operation concurrently is not thread safe and will result in undefined behaviour. This usecase
+   is not supported and might not be supported in future as well. Use multiple instances of
+   librdkafka with different partitions to achieve the same.
```
Comment (emasab) [inline CHANGELOG.md:64]: Maybe add "or use multiple threads to process the batch read by a single thread"

**Review by emasab — CHANGES_REQUESTED** [review]
Please change also the TODO: describe in the release overview

File: CHANGELOG.md (hunk 2 above)
Comment (pranavrth) [reply CHANGELOG.md]: Moved to Introduction.md

**Review by emasab — APPROVED** [review]

**Pull request merged**